platforms/software/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/BuildInitPluginIntegrationTest.groovy

```diff
@@ -402,23 +417,10 @@ Description""") // include the next header to make sure all options are listed
 
         then:
         fails "init"
-        failure.assertHasDescription("Task 'init' not found in project ':some-thing'.")
+        failure.assertHasCause("Aborting build initialization due to existing files in the project directory: '$targetDir'")
```

Comment:
💭 The current failure is an improvement on the previous failure, IMO.  Saying that a built-in task isn't found is strange.